packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -500,6 +520,66 @@ Resources:
             - - Fn::Sub: "arn:${AWS::Partition}:iam::aws:policy/AdministratorAccess"
       RoleName:
         Fn::Sub: cdk-${Qualifier}-cfn-exec-role-${AWS::AccountId}-${AWS::Region}
+      PermissionsBoundary:
+        Fn::If:
+          - DefaultPermissionsBoundary
+          - Ref: PermissionsBoundaryPolicy
+          - Fn::If:
+              - CustomPermissionsBoundary
+              - Fn::Sub: 'arn:${AWS::Partition}:iam:${AWS::Region}:${AWS::AccountId}:policy/${InputPermissionsBoundary}'
+              - Ref: AWS::NoValue
+  PermissionsBoundaryPolicy:
+    Condition: DefaultPermissionsBoundary
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      PolicyDocument:
+        Statement:
+          # If permission boundaries do not have an explicit `allow`, then the effect is `deny`
+          - Sid: ExplicitAllowAll
+            Action:
+              - "*"
+            Effect: Allow
+            Resource: "*"
+          # Add your specific organizational security policy
+          # here, for example denying access to AWS Config
+          - Sid: OrganizationalSecurityPolicy
+            Action:
+              - "config:*"
+            Effect: Deny
+            Resource: "*"
```

Comment:
If we want to leave this here as an example we should comment the entire statement out since we
don't want it to be part of the default policy. I would also
put it at the end.